public/apps/account/account-nav-button.tsx

```diff
@@ -49,7 +49,11 @@ export function AccountNavButton(props: {
   const [modal, setModal] = React.useState<React.ReactNode>(null);
   const horizontalRule = <EuiHorizontalRule margin="xs" />;
   const username = props.username;
-  const [isMultiTenancyEnabled, setIsMultiTenancyEnabled] = React.useState<boolean>(true);
+  const propsMultiTenancyToggle = props.config.multitenancy.enabled;
+  const [isMultiTenancyEnabled, setIsMultiTenancyEnabled] = React.useState<boolean>(
+    propsMultiTenancyToggle || false
+  );
```

Comment:
With this change, if `props.config.multitenancy.enabled` has is `undefined`, do you want `isMultiTenancyEnabled` to be `false`?